pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -775,7 +775,7 @@ func UpdateServiceTemplate(args *commonservice.ServiceTmplObject) error {
 	}
 
 	envStatuses := make([]*commonmodels.EnvStatus, 0)
-	// 去掉检查状态中不存在的环境和主机
+	//remove the status not have env or host
```

Comment:
Use `// Remove environments and hosts that do not exist in the check status`?

Reply:
changed